packages/babel-parser/test/fixtures/typescript/type-arguments/instantiation-expression-binary-operator-2/input.ts

```diff
@@ -0,0 +1 @@
+a<b> > c;
```

Comment:
The current TS nightly throws "Expression expected" in this case and the other similar one, which I think is a TS bug. They should be handled as same as `a<b> ?? c`.

Reply:
It looks like this is intentional: https://github.com/microsoft/TypeScript/pull/49353 (cc @ahejlsberg the code above is unambiguous, do you remember why it's disallowed?).